src/zenml/integrations/deepchecks/validation_checks.py

```diff
@@ -268,9 +298,6 @@ class DeepchecksDataDriftCheck(DeepchecksValidationCheck):
     VISION_NEW_LABELS = source_utils.resolve(
         vision_checks.NewLabels
     ).import_path
-    VISION_TRAIN_TEST_LABEL_DRIFT = source_utils.resolve(
-        vision_checks.TrainTestLabelDrift
-    ).import_path
```

Comment:
There is no available successor of this?

Reply:
It is now renamed to LabelDrift, which I have added in line 289 now. thanks for the careful analysis
```
VISION_LABEL_DRIFT = source_utils.resolve(
        vision_checks.LabelDrift
    ).import_path```